sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -712,7 +712,7 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
         """.stripMargin)
 
       val e = intercept[AnalysisException] {
-        sql("ALTER TABLE tab1 RENAME TO tab2")
+        sql("ALTER VIEW tab1 RENAME TO tab2")
```

Comment:
Will also add a few test cases for ensuring `ALTER TABLE` commands can alter temporary views.